lab/style-script/gss-graphstyle-directive.md

```diff
@@ -0,0 +1,70 @@
+---
+id: gss-graphstyle-directive
+title: Graph Style Script @GraphStyle directive properties
+sidebar_label: GraphStyle directive properties
+---
+
+## `@GraphStyle`
+
+Here is the list of all properties that can be defined in the `@GraphStyle` directive,
+along with their expected types.
+
+### `collision-radius: number`
+
+Sets the margin radius for each node from its centre. If node `size` is `10` and
+`collision-radius` is set to 20, it means there will be `10` spaces left around each
```

Comment:
Why is 10 formatted as code, and 20 is not?

Reply:
Because I am blind :D Will fix it :D